samples/c2pa/C2paPlayer/C2paMenu/C2paMenuFunctions.js

```diff
@@ -98,19 +121,55 @@ export let updateC2PAMenu = function (c2paStatus, c2paMenu , isMonolithic , vide
         );
 
         if (c2paItemValue != null) {
+            //formatting for social media links
+            if(c2paItemKey=== 'SOCIAL' ){
```

Comment:
Is prettier getting run on this file as well? I would expect it to fix the spacing here.